shared/modules/selectors/networks.ts

```diff
@@ -133,7 +133,7 @@ export const getNetworkConfigurationsByCaipChainId = ({
     ([caipChainId, networkConfig]) => {
       const matchesAccount = Object.values(internalAccounts.accounts).some(
         (account) => {
-          const matchesScope = account.scopes?.some((scope) => {
+          const matchesScope = account.scopes.some((scope) => {
```

Comment:
Removing this hotfix from my [OG PR](https://github.com/MetaMask/metamask-extension/pull/36422) (was done to ensure CI was green)